app/templates/about.html

```diff
@@ -9,6 +9,92 @@ <h2>About the POLDER Federated Search App</h2>
     <p>Find out more by reading the <a class="about__link" href="https://polder-crew.github.io/Federated-Search-Documentation/intro.html" rel="noopener" target="_blank">documentation book</a> or exploring the <a class="about__link" href="https://github.com/nein09/polder-federated-search" rel="noopener" target="_blank">GitHub repository</a>.</p>
     <p>If you'd like to send us questions or comments directly, you are welcome to <a class="about__link" href="mailto:cverhey@oceannetworks.ca">email us</a>.</p>
 
+    
+        <div clas="covered_repositories">
+        <h2>Covered data repositories </h2>
+        <p>There are two ways a repository is included in searches that are made from this application- if the repository is indexed DataONE (and has data within the search parameters), and if the repository is indexed by the app itself.</p>
+        <h3>DataONE Repositories</h3>
+        <p>There are many DataONE repositories, but of particular interest to the polar research community are:</p>
+        
+         <div>
+            <a  href="https://arcticdata.io/", rel="noopener", target="_blank">
```

Comment:
In your link tags in here, there are a lot of commas - those should not be there. The ones in the template tags (e.g. `{{ url_for }}`) are fine, because those are actually function parameters, when you get down to it! But the commas inside `<a>` are invalid HTML.

Reply:
Keep the `rel="noopener"` and the `target="_blank"` (those open the links in a new tab), they just should not be separated with commas.